### ISSUE TITLE
Possible problem when installing in Mac OS X

### ISSUE THREAD

**Issue by muammar** [matze/mtheme#211]
Hi, I have done the following:

1. `git clone https://github.com/matze/mtheme`.
2.  `make all`.
3. `make install`.
4. `cd ~/github/mtheme/demo`
5. `xelatex -shell-escape demo.tex`

```
y (/usr/local/texlive/2015/texmf-dist/tex/latex/fontspec/fontspec.sty
(/usr/local/texlive/2015/texmf-dist/tex/latex/l3kernel/expl3.sty
(/usr/local/texlive/2015/texmf-dist/tex/latex/l3kernel/expl3-code.tex)
(/usr/local/texlive/2015/texmf-dist/tex/latex/l3kernel/l3xdvipdfmx.def))
(/usr/local/texlive/2015/texmf-dist/tex/latex/l3packages/xparse/xparse.sty)
(/usr/local/texlive/2015/texmf-dist/tex/latex/fontspec/fontspec-xetex.sty
(/usr/local/texlive/2015/texmf-dist/tex/latex/base/fontenc.sty
(/usr/local/texlive/2015/texmf-dist/tex/latex/euenc/eu1enc.def)
(/usr/local/texlive/2015/texmf-dist/tex/latex/euenc/eu1lmr.fd))
(/usr/local/texlive/2015/texmf-dist/tex/xelatex/xunicode/xunicode.sty
(/usr/local/texlive/2015/texmf-dist/tex/latex/tipa/t3enc.def
(/usr/local/texlive/2015/texmf-dist/tex/latex/euenc/eu1lmss.fd)))
(/usr/local/texlive/2015/texmf-dist/tex/latex/fontspec/fontspec.cfg)))kpathsea:make_tex: Invalid fontname `Fira Mono Medium/OT', contains ' '
kpathsea:make_tex: Invalid fontname `Fira Mono Medium/I/OT', contains ' '
kpathsea:make_tex: Invalid fontname `Fira Mono Medium/OT', contains ' '


!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
!
! fontspec error: "font-not-found"
!
! The font "Fira Mono Medium" cannot be found.
!
! See the fontspec documentation for further information.
!
! For immediate help type H <return>.
!...............................................

l.91 }

?
```

But Fira fonts are indeed installed:

```
muammar@zarathustra ~/github/mtheme/demo [master]
± % locate Fira |grep Fonts                                                                                                                            !10357
/Library/Application Support/Apple/Fonts/iWork Arabic Support/AlFirat.ttc
/Library/Fonts/FiraMono-Bold.otf
/Library/Fonts/FiraMono-Regular.otf
/Library/Fonts/FiraSans-Bold.otf
/Library/Fonts/FiraSans-BoldItalic.otf
/Library/Fonts/FiraSans-Book.otf
/Library/Fonts/FiraSans-BookItalic.otf
/Library/Fonts/FiraSans-Eight.otf
/Library/Fonts/FiraSans-EightItalic.otf
/Library/Fonts/FiraSans-ExtraBold.otf
/Library/Fonts/FiraSans-ExtraBoldItalic.otf
/Library/Fonts/FiraSans-ExtraLight.otf
/Library/Fonts/FiraSans-ExtraLightItalic.otf
/Library/Fonts/FiraSans-Four.otf
/Library/Fonts/FiraSans-FourItalic.otf
/Library/Fonts/FiraSans-Hair.otf
/Library/Fonts/FiraSans-HairItalic.otf
/Library/Fonts/FiraSans-Heavy.otf
/Library/Fonts/FiraSans-HeavyItalic.otf
/Library/Fonts/FiraSans-Italic.otf
/Library/Fonts/FiraSans-Light.otf
/Library/Fonts/FiraSans-LightItalic.otf
/Library/Fonts/FiraSans-Medium.otf
/Library/Fonts/FiraSans-MediumItalic.otf
/Library/Fonts/FiraSans-Regular.otf
/Library/Fonts/FiraSans-SemiBold.otf
/Library/Fonts/FiraSans-SemiBoldItalic.otf
/Library/Fonts/FiraSans-Thin.otf
/Library/Fonts/FiraSans-ThinItalic.otf
/Library/Fonts/FiraSans-Two.otf
/Library/Fonts/FiraSans-TwoItalic.otf
/Library/Fonts/FiraSans-Ultra.otf
/Library/Fonts/FiraSans-UltraItalic.otf
/Library/Fonts/FiraSans-UltraLight.otf
/Library/Fonts/FiraSans-UltraLightItalic.otf
```

I also have installed the package `fira` with `Tex Live Utility`. The fonts in the path `/Library/Fonts` I installed them from here https://github.com/powerline/fonts and they used to work fine with `mtheme` at least before december 2015. Why the `sty` is failing to load the fonts?.

Thanks.

PS. I can provide `diff` files between the *working* sty  I have with respect to the new ones.

**Comment by rchurchley** [comment]
Judging from the error message you are seeing, the Metropolis family is correctly loading the fonts you listed; the only one it can't find is Fira Mono Medium, which is missing from your list of installed fonts. Fira Mono Medium is a fairly recent addition to the Fira font family, added to version 3.108 in June 2014. Updating your fonts to the latest version should fix this problem.

As for why you didn't get this error in previous versions of the Metropolis theme? We recently decided to set the monospace bold font to Fira Mono Medium to match the lighter font weights used elsewhere in the theme. This was originally changed in b47234ba, when we explicitly set the bold and italic fonts to fix a problem (#187) with LuaLaTeX's font handling.

**Comment by muammar** [comment]
Now I feel kind of stupid :). Thanks for your reply. I did this script https://gist.github.com/muammar/a5ffb635eb7f532346a8e777b847f8a7 to install the fonts in mac (it is based on the one in the `docker` dir. I will close this report. Cheers.
